Cookbook/Technical-Documents/NewHiresCheckList.md

```diff
@@ -24,7 +24,6 @@ As an iOS Engineer, you should be in the following Slack channels:
 	- `#ios`
 	- `#ios-build`
 	- `#ios-usa-support`
-	- `#ios-standup`
 	- `#ios-automation`
```

Comment:
Do we need the automation channel? There's been no activity since last September

Reply:
Not sure, I thought about it but since it's not archived I decided to leave it. Happy to remove it if we're not going to use it.

Reply:
:tumbleweed: https://babylonhealth.slack.com/archives/GD6J0QWQ5/p1583145658241800